### ISSUE TITLE
Too big font in setting titles

### ISSUE THREAD

**Issue by VasilisKos** [anilbeesetti/nextplayer#568]
The font size of the setting titles is too big in portrait mode. The result is that setting titles are cut-off, and some of them not easily understandable. That creates problems when translating the app too. The translation text in some languages (like Greek) is even bigger. You have to turn the phone to landscape mode to see them properly. The same is with the explanation text font size of every setting, which is even smaller, but I guess that would be too small if reduced. But the titles font size should definitely be reduced.

**Comment by josephg5** [comment]
I think better to (soft)wrap long lines than trying to reduce font size till everything fits into one line.

**Comment by VasilisKos** [comment]
> I think better to (soft)wrap long lines than trying to reduce font size till everything fits into one line.

That would be even better! I totally agree!

**Comment by advanced03** [comment]
I think the font sizes in general are too big. Even the smallest option is massive

**Comment by U-C-S** [comment]
Let 12 or 10 be the smallest font size for subtitles. Current 15 font size is just too big sometimes.

**Comment by barongi** [comment]
Agreed 100%. The font in the list and grid view is too big but the font in settings in portrait mode is just comically large. It makes the whole app look very unprofessional. Also impossible to translate when only 60% of the text is showing. Change the font to a better/thinner one, choose a smaller size and allow the settings texts under the headers to use 3 rows. The wrong fontsizes all over the place even affect the elements in quick settings.

<img width="656" height="1440" alt="Image" src="https://github.com/user-attachments/assets/3c642f01-509a-48bc-86da-bc451a92b0da" />
